lib/device-log/ios-crash-log.js

```diff
@@ -36,14 +36,18 @@ class IOSCrashLog extends IOSDriverIOSCrashLog {
       log.debug(`Crash reports root '${crashLogsRoot}' does not exist. Got nothing to gather.`);
       return [];
     }
-    const foundFiles = await fs.glob(`${crashLogsRoot}/**/*.crash`, {realpath: true});
+    const foundFiles = await fs.glob(`${crashLogsRoot}/**/*.crash`);
```

Comment:
does this return full paths by default?

Reply:
Yes. Just doesn't run [fs.realpath](https://nodejs.org/api/fs.html#fs_fs_realpath_path_options_callback) on every entry.